src/x11/x.c

```diff
@@ -53,35 +53,44 @@ cairo_surface_t *x_create_cairo_surface(void)
 
 void x_win_move(int width, int height)
 {
+        // Previous dimensions of the window to avoid calling X11 if no change
+        // is needed
+        static struct {
```

Comment:
Why not `static struct dimension window_dim = { 0 }`?

Reply:
`struct dimension` is only visible in `draw.c`. I'm trying to avoid having a lot of gobally available structs. I'll refactor this part more in subsequent commits anyway, I plan to create our own window data structure to allow handling of multiple windows in the future.